include/seqan3/core/platform.hpp

```diff
@@ -74,9 +74,9 @@
 #        pragma GCC warning "Be aware that GCC < 11.3 might have bugs that cause SeqAn3 fail to compile."
 #    endif // (__GNUC__ == 11 && __GNUC_MINOR__ <= 2)
 
-#    if (__GNUC__ == 12 && __GNUC_MINOR__ <= 1)
-#        pragma GCC warning "Be aware that GCC < 12.2 might have bugs that cause SeqAn3 fail to compile."
-#    endif // (__GNUC__ == 12 && __GNUC_MINOR__ <= 1)
+// #    if (__GNUC__ == 12 && __GNUC_MINOR__ <= 1)
+// #        pragma GCC warning "Be aware that GCC < 12.2 might have bugs that cause SeqAn3 fail to compile."
+// #    endif // (__GNUC__ == 12 && __GNUC_MINOR__ <= 1)
```

Comment:
I think you already said to me why, but can your repeat?

Reply:
It's in the commit comment :D
```
Ubuntu CI still uses 12.1. Since 12.1 is the first release, there is no point in checking gcc 12
```